pyproject.toml

```diff
@@ -73,3 +73,26 @@ line_length = 79
 profile = "black"
 skip_gitignore = "True"
 verbose = "False"
+
+[tool.repo-review]
+# These are a list of the currently failing tests:
```

Comment:
Thanks @ukmo-ccbunney for your first CF-Units PR! I'm happy with the changes, only thing I could ask of you is to create an issue with a tasklist to consider/address the currently failing repo-review tests. (similar tasklist to this one scitools/iris#5780)

Reply:
Done! See https://github.com/SciTools/cf-units/issues/462